content/academy/data_collection_basics/postman.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Postman
+description: Learn about Postman, a simple tool for testing requests and proxies
```

Comment:
It's good for SEO to have the meta descriptions between 120 and 160 characters long. If you can wizard up some longer descriptions for these files, that would be awesome ❤️